### ISSUE TITLE
Investigate Possible Roll20 HP Sync Issue

### ISSUE THREAD

**Issue by Aeristoka** [kakaroto/Beyond20#964]
Not entirely sure what's going on with the report, but as reported here: https://discord.com/channels/578874022968688640/579077106173083659/957297425628938300

Possible issue that causes Roll20 to not properly receive HP Sync from DnDBeyond via Beyond20, potentially when multiple tokens exist for the same character (even on other scenes?)

**Comment by Aeristoka** [comment]
Potentially also this? https://discord.com/channels/578874022968688640/579077106173083659/957297425628938300

**Comment by Lindharin** [comment]
We may have had this issue in our game today.  Multiple characters took damage, we applied the damage to our character sheets in D&D Beyond, but the tokens (and character sheet) were not updated in Roll20.  We tried refreshing both Roll20 and D&D Beyond, making additional HP adjustments, etc.  Nothing we did in D&D Beyond would sync to Roll20's hit point fields. 
 All the affected characters did have temporary hit points at the time.  We manually edited the tokens in Roll20 to remove the temporary hit points and inflict the remaining damage.   On the next turn, we took more damage, applied it in D&D Beyond, and the tokens were correctly updated in Roll20.  Since then tokens have been updating correctly, whether or not we have temporary hit points at the time.   We have not been able to reproduce the issue since then.
